packages/protoplugin/src/create-es-plugin.ts

```diff
@@ -313,6 +329,7 @@ function splitParameter(
     return {
       key: i === -1 ? pair : pair.substring(0, i),
       value: i === -1 ? "" : pair.substring(i + 1),
+      raw: pair,
```

Comment:
Small code style nit: Can be

```suggestion
      raw,
```

if you rename the variable `pair`.